Tests/TestingApps/PurchaseTesterSwiftUI/PurchaseTester.xcodeproj/project.pbxproj

```diff
@@ -971,8 +971,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/RevenueCat/purchases-ios";
 			requirement = {
-				kind = upToNextMajorVersion;
```

Comment:
Oh I guess depending on when the tag was created this could have found the last release. But this PR will be better, allowing us to use the job from any branch too.